### PR TITLE
Fix spelling errors.

### PR DESCRIPTION
**Pull request by gspencergoog**
Fix the spelling errors in the dartdocs for the framework.

There are no functionality changes here, just documentation fixes.

### CODE REVIEW

**Comment by eseidelGoogle** [comment]
What magic did you use to find these?!? 😄

**Comment by gspencergoog** [comment]
A couple of magic scripts that I wrote a while ago (the last time I did this), combined with `ispell`.

**Review by kf6gpe — APPROVED** [review]
Thanks!! LGTM.

**Review by Piinks — APPROVED** [review]
LGTM!

**Comment by cbracken** [comment]
Please file a bug for us to make this a presubmit.

**Comment by gspencergoog** [comment]
Heh.  Making this a presubmit would be pretty flaky, I think.  It's kind of a manual process, and we'd have to keep some kind of "exception" list or something, which could get tedious.

**Comment by cbracken** [comment]
SGTM. I have a small dart script in a three-year-old fork somewhere from the last time I did this that goes the other way -- rather than a blanket list, it just had a list of common corrections that I sourced from the UK->US equivalents and common misspellings pages on Wikipedia. I can see if it's still in usable shape. The biggest issue at the time was it was using my own cheapo parser, not the analyzer.

**Comment by gspencergoog** [comment]
Mine doesn't use the analyzer either, but It's pretty easy to just look at the dartdocs.  I do things like skip over example code (but not the descriptions), and things in square brackets, which cuts down on a lot of the things the spelling checker flags.  If I wanted to improve it, I would probably make it skip over URLs and paths too.  Still, there are plenty of places where it required a human to judge.

I do like the idea of just looking for common misspellings, that would probably catch a lot of them.

File: packages/flutter/lib/src/services/platform_views.dart [hunk 1]
```diff
@@ -349,12 +349,12 @@ class AndroidMotionEvent {
...
   final int deviceId;
 
-  /// A bitfield indicating which edges, if any, were touched by this MotionEvent.
+  /// A bit field indicating which edges, if any, were touched by this MotionEvent.
   ///
   /// See Android's [MotionEvent#getEdgeFlags](https://developer.android.com/reference/android/view/MotionEvent.html#getEdgeFlags()).
   final int edgeFlags;
 
-  /// The source of this event (e.g a touchpad or stylus).
+  /// The source of this event (e.g a touch pad or stylus).
```
Comment (jamesderlin) [inline packages/flutter/lib/src/services/platform_views.dart]: IMO we should leave this one alone.
https://www.dictionary.com/browse/touchpad?s=t
https://en.wikipedia.org/wiki/Touchpad

On my Linux machine, the configuration for this is under "Mouse and Touchpad".

File: packages/flutter/lib/src/services/text_formatter.dart [hunk 2]
```diff
@@ -16,7 +16,7 @@ import 'text_input.dart';
 ///
 /// Concrete implementations [BlacklistingTextInputFormatter], which removes
 /// blacklisted characters upon edit commit, and
-/// [WhitelistingTextInputFormatter], which only allows entries of whitelisted
+/// [WhitelistingTextInputFormatter], which only allows entries of white-listed
```
Comment (jamesderlin) [inline packages/flutter/lib/src/services/text_formatter.dart]: I think we should leave these alone too.
https://www.dictionary.com/browse/whitelist?s=t
https://en.wikipedia.org/wiki/Whitelisting

I think treating it as a single non-hyphenated word also would be more consistent with WhitelistingTextInputFormatter.

Furthermore, "blacklist" is left alone elsewhere.

Comment (gspencergoog) [reply packages/flutter/lib/src/services/text_formatter.dart]: OK, sure.  Reverted.

File: packages/flutter/lib/src/services/platform_views.dart (hunk 1 above)
Comment (gspencergoog) [reply packages/flutter/lib/src/services/platform_views.dart]: OK, reverted.

**Review by jamesderlin — APPROVED** [review]
LGTM.  Thanks!

**Pull request merged**